SConstruct

```diff
@@ -1019,6 +1019,9 @@ if env['coverage']:
 if env['toolchain'] == 'mingw':
     env.Append(LINKFLAGS=['-static-libgcc', '-static-libstdc++'])
 
+if env["OS"] == "Windows":
+    env.Append(CPPDEFINES=["CT_DLL_EXPORT"])
```

Comment:
I am assuming that this appends to `CPPDEFINES`.

Reply:
I think you only want to set this while compiling the library sources, e.g. from the `localenv` that's created in `src/SConscript`. Setting it here will affect other cases like the tests, which is not what you want.